### DISCUSSION TITLE
kukui shim not work

### DISCUSSION THREAD

**Discussion by psannetwork** [3kh0/ext-remover#1329] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello,
I am using a Chromebook with the board name "kukui."
On this board, I am unable to boot the Shim.
Both the Raw Shim and Legacy Shim hang during the boot process.

I would like to use Br1ck, but I’m not sure how to proceed. Could you please advise me on what to do?

Here are the details of the hang:

I wrote the Shim to a USB drive.
After entering the developer mode block screen, I transitioned to recovery mode and inserted the USB.
At this point, the screen blacks out.
Thank you for your help.

**Comment by schoolexploitkid** [comment, marked as answer]
dm @ubyte on the titanium network discord (https://discord.gg/unblock)

**Comment by ghost** [comment]
you're probably keyrolled
